### PR TITLE
Add integration tests to Trello::Card

### PR DESCRIPTION
**Pull request by wokenqingtian**
Base on #282 Better merge it first.
Add integration tests(or unit tests) for all Trello::Card methods
  - [x] Trello::Card.find
  - [x] Trello::Card.create
  - [x] Trello::Card.path_name
  - [x] Trello::Card#update_fields
  - [x] Trello::Card#actions
  - [x] Trello::Card#board
  - [x] Trello::Card#cover_image
  - [x] Trello::Card#checklists
  - [x] Trello::Card#plugin_data
  - [x] Trello::Card#custom_field_items
  - [x] Trello::Card#check_item_states
  - [x] Trello::Card#list
  - [x] Trello::Card#members
  - [x] Trello::Card#voters
  - [x] Trello::Card#save
  - [x] Trello::Card#update!
  - [x] Trello::Card#delete
  - [x] Trello::Card#closed?
  - [x] Trello::Card#close
  - [x] Trello::Card#close!
  - [x] Trello::Card#valid?
  - [x] Trello::Card#add_comment
  - [x] Trello::Card#add_checklist
  - [x] Trello::Card#create_new_checklist
  - [x] Trello::Card#move_to_list
  - [x] Trello::Card#move_to_list_on_any_board
  - [x] Trello::Card#move_to_board
  - [x] Trello::Card#add_member
  - [x] Trello::Card#remove_member
  - [x] Trello::Card#upvote
  - [x] Trello::Card#remove_upvote
  - [x] Trello::Card#add_label
  - [x] Trello::Card#remove_label
  - [x] Trello::Card#add_attachment
  - [x] Trello::Card#attachments
  - [x] Trello::Card#remove_attachment
  - [x] Trello::Card#comments
  - [x] Trello::Card#created_at

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,22 +11,23 @@ Seriously, [check it out](http://www.trello.com/).
...
-```
-
 Full Disclosure: This library is mostly complete, if you do find anything missing or not functioning as you expect it
-to, please [let us know](https://trello.com/card/spot-a-bug-report-it/4f092b2ee23cb6fe6d1aaabd/17).
+to, please just create an issue.
 
-Supports Ruby 2.1.0 or newer.
+## Requirements
 
+Use the newest version for Ruby 2.1.0 or newer support.
```
Comment (runlevel5) [inline README.md:19]: I'd like to propose that we only support Ruby 2.5 or newer. Ruby 2.1.0 has reached its EOL for quite a while.

File: README.md [hunk 2]
```diff
@@ -12,22 +11,23 @@ Seriously, [check it out](http://www.trello.com/).
...
 
-## Installation
-
-```
-# gem install ruby-trello
-```
-
 Full Disclosure: This library is mostly complete, if you do find anything missing or not functioning as you expect it
-to, please [let us know](https://trello.com/card/spot-a-bug-report-it/4f092b2ee23cb6fe6d1aaabd/17).
+to, please just create an issue.
```
Comment (runlevel5) [inline README.md:15]: Could we add the link to Github Issue here please?

Comment (wokenqingtian) [reply README.md:15]: Sure. I have added it in #278

File: README.md (hunk 1 above)
Comment (wokenqingtian) [reply README.md:19]: I like this. I have changed them in #278. And what version of Rails do you think we should support?

**Comment by jeremytregunna** [comment]
Is this good to go? There's an unanswered question above, please advise.

File: README.md (hunk 1 above)
Comment (runlevel5) [reply README.md]: I think we should support 5.2 and 6.x

**Comment by runlevel5** [comment]
> Is this good to go? There's an unanswered question above, please advise.
@jeremytregunna it is good to me

**Comment by jeremytregunna** [comment]
Perfect, thanks @runlevel5

File: README.md (hunk 1 above)
Comment (wokenqingtian) [reply README.md]: 👌

**Comment by wokenqingtian** [comment]
🎉

**Pull request merged**